src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -219,14 +213,19 @@ private void setGoalPosition(double height, double armOffset) {
     logGoalVelocity.append(0);
   }
 
-  /** Returns whether the elevator is at goal position. */
-  public boolean atGoalPosition() {
-    return MathUtil.isNear(goalState.position, getHeight(), GOAL_POSITION_TOLERANCE);
+  /** Returns whether the elevator is at goal height. */
+  public boolean atGoalHeight() {
+    return MathUtil.isNear(goalState.position, getHeight(), GOAL_HEIGHT_TOLERANCE);
+  }
+
+  /** Returns whether the elevator is above the height where it is safe to tip the arm outward. */
+  public boolean isAboveSafeArmPivotHeight() {
+    return currentState.position >= goalState.position - armPivotHeightOffset;
   }
 
-  /** Returns whether the elevator is above the arm position. */
-  public boolean aboveArmPosition() {
-    return goalState.position - armOffset <= currentState.position;
+  /** Returns whether the elevator is near the given {@link ElevatorLevel}. */
+  public boolean isNearestToLevel(ElevatorLevel level) {
+    return MathUtil.isNear(level.getElevatorHeight(), getHeight(), 0.1);
```

Comment:
Should we use multiple of the `GOAL_POSITION_TOLERANCE`? 2x, 3x? Does it need to be 10cm?